metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java

```diff
@@ -78,115 +70,6 @@ private static URL checkIfUrlIsValid(HashSet<Report> reports, String id) {
     }
   }
 
-  private static HttpURLConnection getClient(URL checkedUrl) throws IOException {
```

Comment:
There is a constant `public static final String HTTPS = "https";` that is no longer used.

Reply:
removed.